### ISSUE TITLE
Use addresses of contacts as destination and start location

### ISSUE THREAD

**Issue by sedrubal** [grote/Transportr#230]
In öffi you can select the addresses of contacts from address book as destination and as start location for routes. This would be a nice feature, too.

**Comment by grote** [comment]
I agree, but many people will complain why Transportr needs to access the address book. I personally don't consider it an important feature, but would accept a pull request that adds it cleanly.

**Comment by sedrubal** [comment]
Yes, you're right. But in android 6 you can switch off permissions, so this might be OK. Unfortunately I can't code android apps, so anyone else has to do the PR ;)

**Comment by patkan** [comment]
Apart from that this feature would be indirectly implemented by #200.
With this issue you could go to your contacts and pass on the address to Transportr with the navigation button.

**Comment by grote** [comment]
If #200 would allow for that, it would be the preferable and easier way to implement this feature.

**Comment by grote** [comment]
When clicking on an address in the address book, I get an intent like that `geo:0,0?q=my+street+address`.

Then we would still need to find out somehow how to get the coordinates for `my+street+address`.
